lib/news_scraper/transformers/article.rb

```diff
@@ -36,8 +36,9 @@ def transformed_response
       def parsed_data(data_type)
         return nil unless scrape_details[data_type]
 
-        scrape_pattern = scrape_details[data_type]['pattern']
         scrape_method = scrape_details[data_type]['method'].to_sym
+        scrape_pattern = scrape_details[data_type]['pattern']
+        scrape_pattern = "(#{scrape_pattern})[1]" if scrape_method == :xpath
```

Comment:
ya http://stackoverflow.com/questions/3319341/why-do-indexes-in-xpath-start-with-1-and-not-0